test/unit/utils/utils.js

```diff
@@ -1,9 +1,29 @@
+const load = require;
+
 function isFieldDuplicated(field) {
-  return function isDuplicated(field1, field2) {
-    return field1.CaseTypeID === field2.CaseTypeID
-      && field1[field] === field2[field]
-      && field1.UserRole === field2.UserRole;
-  };
+    return function isDuplicated(field1, field2) {
+        return field1.CaseTypeID === field2.CaseTypeID
+            && field1[field] === field2[field]
+            && field1.UserRole === field2.UserRole;
+    };
+}
+
+function loadAllFilesIn(location, featureFiles) {
+    return function loadFeatureFiles()
+    {
+        let definitions = [];
+
+        featureFiles.forEach(featureFile => {
+            definitions = definitions
+                .concat(load(`definitions/divorce/json/${location}/${featureFile}.json`));
+        });
+
+        return definitions;
+    }
```

Comment:
@yarpo How do you see this a function to enable us load the feature files by supplying the folder and list of files we want loaded. I had to give the function a name as sonar didn't like anonymous function definition. This was the require works as suggested earlier and we can reuse this in other places when need arises

Reply:
@jvictorcf Any thoughts?

Reply:
IS:
```function loadAllFilesIn(location, featureFiles) { // HERE!
    return function loadFeatureFiles()
    {
        let definitions = [];

        featureFiles.forEach(featureFile => {
            definitions = definitions
                .concat(load(`definitions/divorce/json/${location}/${featureFile}.json`));
        });

        return definitions;
    }
}
```

Should be:
```function loadAllFilesIn(location) { // HERE 1
    return function loadFeatureFiles(featureFiles)  // HERE 2
    {
        let definitions = [];

        featureFiles.forEach(featureFile => {
            definitions = definitions
                .concat(load(`definitions/divorce/json/${location}/${featureFile}.json`));
        });

        return definitions;
    }
}
```

Reply:
then call

```
const loadAllFiles = loadFeatureFiles('blabla/AuthCaseBlabval'); // `loadAllFiles` is now a reference on function

const allDefinitions = loadAllFiles([
   'file-a',    'file-b',   'file-c'
]);
```